demos/typescript-app/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "typescript-app",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack --watch"
```

Comment:
I think there is no test script and no need of this script.
About build, please unify with esm-demo-app way. Or do you think this setting is good for TS tryout?

Reply:
I will unify with esm-demo-app